core/domain/suggestion_registry.py

```diff
@@ -825,14 +833,20 @@ def convert_html_in_suggestion_change(
             conversion_fn: function. The function to be used for converting the
                 HTML.
         """
-        # Here, change is of type ExplorationChange and all attributes
-        # on ExplorationChange are created dynamically except cmd, so due
-        # this MyPy is unable to recognize `content_html` and `translation_html`
+        # Here we use MyPy ignore because change is of type ExplorationChange
```

Comment:
Ditto here and below.

Reply:
This issue is already fixed in https://github.com/oppia/oppia/pull/15958